src/main/java/Gutti/FindCommand.java

```diff
@@ -0,0 +1,74 @@
+package Gutti;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a command that searches for tasks with descriptions containing a specified word.
+ * <p>
+ * This class extends {@code Command} and provides functionality to search through a list of tasks
+ * and output tasks that match the search criteria.
+ * </p>
+ */
+public class FindCommand extends Command{
+    /** The word to match in the task descriptions. */
+    public String wordToMatch;
+
+    /**
+     * Constructs a {@code FindCommand} with the specified word to match.
+     *
+     * @param wordToMatch The word to search for in task descriptions.
+     */
+    public FindCommand (String wordToMatch) {
+        this.wordToMatch = wordToMatch;
+    }
+
+    /**
+     * Executes the find command to search through the list of tasks for descriptions containing the specified word.
+     * <p>
+     * It iterates through the list of tasks, checks if the description of each task contains the search word,
+     * and collects the indices of matching tasks. It then prints out the matching tasks with their indices.
+     * If no tasks are found with the specified word, it prints a message indicating that no matching tasks were found.
+     * </p>
+     *
+     * @param tasks The {@code TaskList} containing the list of tasks to search through.
+     * @param ui The {@code Ui} instance for interacting with the user interface (not used in this method).
+     * @param storage The {@code Storage} instance for saving and loading tasks (not used in this method).
+     * @throws GuttiException If an error occurs during the execution of the command.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws GuttiException {
+        ArrayList<Task> taskList = tasks.getTasks();
+        ArrayList<Integer> integerList = new ArrayList<>();
+
+        for (int i = 0; i < taskList.size(); i++) {
+            Task specificTask = taskList.get(i);
+            if (specificTask.description.toLowerCase().contains(wordToMatch.toLowerCase())) {
+                integerList.add(i);
+            }
+        }
+        if (!integerList.isEmpty()) {
+            System.out.println("____________________________________________________________");
```

Comment:
Could put the line into a variable  instead of always having to print the line as the line is always the same.